src/utils/escape.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+const escapeRegex = /[[\].#*$><+~=|^:(),"'`]/g
```

Comment:
Since we’re already here, let’s add a dash to this chat group and add a plus to the end, so we can get rid of the multi dash regex below

Reply:
I don't think we can since the dash replacement itself happens as a result of that regex

Reply:
We can 😉

.replace(/[-()]+/g, '-')

At least I’m 99% sure... left some things out above since I’m on my phone

Reply:
hmm ok I'll give it a try